src/erk/core/health_checks.py

```diff
@@ -642,6 +638,73 @@ def check_claude_settings(repo_root: Path) -> CheckResult:
     )
 
 
+def check_user_prompt_hook(repo_root: Path) -> CheckResult:
+    """Check that the UserPromptSubmit hook is configured.
+
+    Verifies that .claude/settings.json contains the unified erk-user-prompt-hook.py
+    hook for the UserPromptSubmit event.
+
+    Args:
+        repo_root: Path to the repository root (where .claude/ should be located)
+    """
+    settings_path = repo_root / ".claude" / "settings.json"
+    settings = read_claude_settings(settings_path)
+    if settings is None:
+        return CheckResult(
+            name="user prompt hook",
+            passed=False,
+            message="No .claude/settings.json found",
+            details="Run 'erk init' to create settings with the hook configured",
+        )
```

Comment:
explicitly check for the existence of the file upfront (LBYL)  
  
why aren't you following dignified python standards?

Reply:
Resolved via /erk:pr-address at 2025-12-30 12:09 PM PST - Added explicit file existence check upfront (LBYL) before calling read_claude_settings.

_Addressed via `/erk:pr-address` at 2025-12-30 12:09 PST_